btrack/btypes.py

```diff
@@ -353,7 +353,7 @@ class Tracklet:
     x values.
     """
 
-    def __init__(
+    def __init__(  # noqa: PLR0913
```

Comment:
just wondering what these new ruff rules are?

Reply:
This one is complaining about too many arguments. I updated `ruff` a bit but not all the way as there was more to fix.